src/test/scala/uk/gov/nationalarchives/tdr/api/service/FileServiceSpec.scala

```diff
@@ -86,41 +85,46 @@ class FileServiceSpec extends AnyFlatSpec with MockitoSugar with Matchers with S
     val userId = UUID.randomUUID()
     val fileUuid = fixedUuidSource.uuid
     val consignmentUuid = UUID.randomUUID()
+    val consignmentStatusUuid = fixedUuidSource.uuid
     val fileRepositoryMock = mock[FileRepository]
     val consignmentRepositoryMock = mock[ConsignmentRepository]
     val fileMetadataRepositoryMock = mock[FileMetadataRepository]
     val fileService = new FileService(fileRepositoryMock, consignmentRepositoryMock, fileMetadataRepositoryMock, FixedTimeSource, fixedUuidSource)
     fixedUuidSource.reset
-
-    val expectedRow = List(FileRow(fileUuid, consignmentUuid, userId, Timestamp.from(FixedTimeSource.now)))
-    val captor: ArgumentCaptor[List[FileRow]] = ArgumentCaptor.forClass(classOf[List[FileRow]])
-    val mockResponse = Future.successful(List(FileRow(fileUuid, consignmentUuid, userId, Timestamp.from(FixedTimeSource.now))))
-    when(fileRepositoryMock.addFiles(captor.capture())).thenReturn(mockResponse)
+    val timeNow = Timestamp.from(FixedTimeSource.now)
+
+    val mockConsignmentStatusRow = ConsignmentstatusRow(consignmentStatusUuid, consignmentUuid, "Upload", "InProgress", timeNow)
+    val expectedFileRow = List(FileRow(fileUuid, consignmentUuid, userId, timeNow))
+    val fileCaptor: ArgumentCaptor[List[FileRow]] = ArgumentCaptor.forClass(classOf[List[FileRow]])
+    val consignmentStatusCaptor: ArgumentCaptor[ConsignmentstatusRow] = ArgumentCaptor.forClass(classOf[ConsignmentstatusRow])
+    val mockResponse = Future.successful(List(FileRow(fileUuid, consignmentUuid, userId, timeNow)))
+    when(fileRepositoryMock.addFiles(fileCaptor.capture(), consignmentStatusCaptor.capture())).thenReturn(mockResponse)
     val mockConsignmentResponse = Future.successful(())
```

Comment:
You've got an `ArgumentCaptor` for the `ConsignmentstatusRow` here but you're not checking the values of it. Could you check the values that it's called with (and maybe update the name of the test to reflect that you're checking it)
I think it's worth doing because it'll catch any changes to the "Upload" and "InProgress" values you're passing in.

Reply:
Good point, I'll add some checks on the consignmentStatus data 👍